core/src/main/java/io/kestra/core/tasks/flows/WorkingDirectory.java

```diff
@@ -271,8 +272,11 @@ public void preExecuteTasks(RunContext runContext, TaskRun taskRun) {
                 // Set the cacheDownloadedTime so that we can check if files has been updated later
                 cacheDownloadedTime = System.currentTimeMillis();
             }
-        } catch (IOException e) {
-            runContext.logger().error("Unable to execute WorkingDirectory pre actions", e);
```

Comment:
Cache was designed to be fail-tolerant, if you didn't succeed in caching an error is displayed but the task is still runs. You change the behaviour here and I'm not sure it's a good idea as the task can still run even if it fails to use its cache.

Reply:
Cache should never failed, I don't see a reason to let failed a cache failed silently and even more that the error will be on the worker and not on the user side (ui)